pkg/auth/providers/oidc/oidc_provider.go

```diff
@@ -537,6 +538,7 @@ func ConfigToOauthConfig(endpoint oauth2.Endpoint, config *v32.OIDCConfig) oauth
 func (o *OpenIDCProvider) getGroupsFromClaimInfo(claimInfo ClaimInfo) []v3.Principal {
 	var groupPrincipals []v3.Principal
 
+	// GitLab uses the full_group_path claim
```

Comment:
I would drop this comment, if it's not Gitlab specific. IIRC Keycloak is using this claim.